frappe/custom/doctype/customize_form/customize_form.py

```diff
@@ -454,6 +454,44 @@ def reset_to_defaults(self):
 		reset_customization(self.doc_type)
 		self.fetch_to_customize()
 
+	def validate_series(self, autoname=None, name=None):
```

Comment:
Why not import this function from `doctype.py`?

Reply:
Yes, will do.

Reply:
Also check usage of `validate_series`. It is used as an object method in erpnext, in `naming_series.py`

Reply:
@prssanna will create PR for that too.